### ISSUE TITLE
Agent cannot find start event for end event for HTTP calls made by Elasticsearch .NET client

### ISSUE THREAD

**Issue by SergeyKleyman** [elastic/apm-agent-dotnet#500]
Origin: https://discuss.elastic.co/t/apm-server-error-connection-refused/200109/3

**Comment by JoseIgnacioD** [comment]
I have the same warning: "**Failed capturing request (failed to remove from ProcessingRequests) - This Span will be skipped in case it wasn't captured before**". Is there any way to silence this warning?  Change the log level of the APM .NET Agent doesn't worked.

**Comment by dhtek** [comment]
Exactly the same warning. 

I tried this code in appsettings.json: 
```
"ElasticApm": {
    "LogLevel": "Error",
  },
```
and this one :

```
"Logging": {
    "LogLevel": {
      "Default": "Debug",
      "Elastic.Apm": "Error"
    }
  },
```

but it's not working.

Can someone could help on this issue ?

**Comment by Larswa** [comment]
Hi @JoseIgnacioD   I am seeing the same problem, and I had to add the `"Elastic.Apm": "Error"` to the Serilog logging level section. That worked for me. No longer getting the span warnings logged.

```
"Serilog": {
      "MinimumLevel": {
        "Default": "Information",
        "Override": {
          "Microsoft": "Information",
          "System": "Information",
          "Elastic.Apm": "Error"

```

**Comment by connorads** [comment]
I'm also experiencing this issue
- netcoreapp2.1
- Serilog.Sinks.Elasticsearch 8.0.0
- Elastic.Apm.NetCoreAll 1.2.0

Thanks for the workaround @Larswa

Anyone found the route cause? 🧐

**Comment by andrewbanchich** [comment]
I'm also experiencing this.

**Comment by elvishsu66** [comment]
When Serilog.Sinks.Elasticsearc is used, the Span is no longer captured?

**Comment by jhtong** [comment]
I am also experiencing this issue.

**Comment by gregkalapos** [comment]
I tried to reproduce this on `master` -  no success so far. 

I strongly think this was fixed in https://github.com/elastic/apm-agent-dotnet/issues/734 which was released in version `1.3.1`.

Can someone please confirm that this issue still exists or if it was fixed? `1.4` is the latest, that should be also have the fix. 

Versions older than `1.3.1` should be affected, since `1.3.1` this should be fixed.

**Comment by gregkalapos** [comment]
Since I received no response and I assume this is fixed - I'm closing this now. Please reopen if this still happens in versions `1.3.1` or newer.
